ophyd/areadetector/plugins.py

```diff
@@ -86,8 +89,9 @@ def __init__(self, *args, **kwargs):
     port_name = C(EpicsSignalRO, 'PortName_RBV', string=True)
 
     def stage(self):
-        # Ensure the plugin is enabled. We do not disable it on unstage.
-        set_and_wait(self.enable, 'Enable')
+        if self.auto_enable:
+            # Ensure the plugin is enabled. We do not disable it on unstage.
+            set_and_wait(self.enable, 'Enable')
```

Comment:
If this was disabled to begin with, it should be disabled on unstage, a la 'stage_sigs'.